cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -177,8 +177,6 @@ def serialize_certificate(certificate):
             "name": certificate_data['name'],
             "description": certificate_data['description'],
             "version": CERTIFICATE_SCHEMA_VERSION,
-            "org_logo_path": certificate_data.get('org_logo_path', ''),
-            "is_active": certificate_data.get('is_active', False),
```

Comment:
Are you using .get() with a default because this was a schema modification and previous data will not have these keys?